kia-lib/src/test/kotlin/com/faforever/ice/gpgnet/GpgnetProxyTest.kt

```diff
@@ -56,6 +57,7 @@ class GpgnetProxyTest {
     @AfterEach
     fun afterEach() {
         sut.close()
+        clearAllMocks()
```

Comment:
The existing test below calls `mockExecutorSingleThreaded()` which needs to be cleared for the other test further down.  These tests ran fine until I renamed the one below and their execution order changed.  Took me a minute to figure this one out.